src/lib/ledgers/five-bells-ledger.js

```diff
@@ -146,18 +134,24 @@ FiveBellsLedger.prototype._autofund = function * () {
       connector: config.getIn(['server', 'base_uri']),
       password: this.credentials.password,
       fingerprint: this.credentials.fingerprint
-    },
-    cert: admin.cert,
-    ca: admin.ca,
-    key: admin.key
-  }, 'could not create account at ledger ' + this.id)
+    }
+  }, 'could not create account at ledger ' + this.id, admin)
 }
 
-function * request_retry (opts, error_msg) {
+function * request_retry (opts, error_msg, credentials) {
   let delay = backoffMin
   while (true) {
     try {
-      let res = yield request(opts)
+      let res = yield request(lodash.defaults(opts, {
+        auth: credentials && {
```

Comment:
In the config we take care of the case missing username/password/key, https://github.com/interledger/five-bells-connector/blob/1c58fa9d99c0e57d6fcfb4fa0eefa3b165233534/src/lib/config.js#L123-L128